### DISCUSSION TITLE
Setup Date and Time

### DISCUSSION THREAD

**Discussion by kentoR32** [brutella/hkknx-public#190] [Q&A]
Hey,

i just freh install the homebridge via the image on a new sd card.
after installation and the first boot i see that the date and the time on the Pi is false.

[1.3.0 ](2021-05-20T12:24:19+02:00)

How can i change it? I search on the WWW but no one of the guides work for me ... 

Thanks a lot

**Comment by brutella** [comment]
If your Raspberry Pi is connected to the internet, the date and time should automatically be updated on a system restart. Could you please verify that.

**Reply by kentoR32** [reply]
Hey,

thanks for the quick reply.

the Pi is connected with ethernet to the www. (ping tested)
i restarted the pi a few times but nothing changes, the date and time is still (2021-05-20T12:24:19+02:00)
